apps/nginx/Dockerfile

```diff
@@ -28,8 +27,10 @@ RUN mkdir -p /opt/var/cache/nginx && \
     cp -a --parents /usr/lib/x86_64-linux-gnu/libcrypto.so.* /opt && \
     cp /usr/share/zoneinfo/${TIME_ZONE:-ROC} /opt/etc/localtime
 
-FROM gcr.io/distroless/base-debian10:nonroot
+FROM gcr.io/distroless/base-debian11:nonroot
+
 COPY --from=base /opt /
+COPY --from=base /usr/lib/x86_64-linux-gnu/libpcre2-8.so.0 /usr/lib/x86_64-linux-gnu/
```

Comment:
@ng-htec Is it possible to move this COPY cmd in a previous stage?

Reply:
@jz-htec We could copy this in previous stage and it will look like

`cp -a --parents /usr/lib/x86_64-linux-gnu/libpcre2-8.so.* /opt && \`

And then the next stage will be

`COPY --from=base /opt/usr/lib/x86_64-linux-gnu/libpcre2-8.so.* /usr/lib/x86_64-linux-gnu/`

Reply:
I think you don't need COPY command in the second stage because the `cp` in the first stage will copy all necessary files in  /opt/* dir, so the COPY layer in the second stage will catch. But, let's test this and see. 👍🏽

Reply:
@jz-htec you are correct, it does work like that, I just have tested it. 

I will push the update now.